pnpm-lock.yaml

```diff
@@ -1755,7 +1757,6 @@ packages:
       express: 4.17.1
       untildify: 4.0.0
     transitivePeerDependencies:
-      - '@prisma/client'
```

Comment:
Unrelated change? Or what causes this?

Reply:
Unrelated, it happens sometimes when running the setup script or when using `pnpm install` in one of the packages (which the setup script essentially does for each package separately). Might be a pnpm bug.

It doesn't really seem to influence anything whether this line is present in the lockfile or not, and even when this change is committed, it returns soon in one of the renovate PRs anyway. So there's no harm when this happens, it's just slightly annoying, but not annoying enough that someone took time and investigated or prepared a bug report for pnpm 😄